src/cco-modules/InformationEntityOntology.ttl

```diff
@@ -367,18 +389,21 @@ cco:ont00001980 rdf:type owl:ObjectProperty ;
                  owl:propertyChainAxiom ( obo:BFO_0000178
                                           cco:ont00001982
                                         ) ;
+                 rdfs:domain cco:ont00000127 ;
+                 rdfs:range obo:BFO_0000001 ;
                  rdfs:label "describes condition"@en ;
-                 skos:definition "p describes_condition c iff p is an instance of a Performance Specification, and p has part d, and d is an instance of a Descriptive Information Content Entity, and p prescribes an entity s, and d describes c, and c is an entity that is causally relevant to s existing as prescribed by p."@en ;
+                 skos:definition "p describes_condition c iff p is an instance of a Performance Specification and c is an instance of Entity and p has part d, and d is an instance of a Descriptive Information Content Entity, and p prescribes an entity s, and d describes c, and c is an entity that is causally relevant to s existing as prescribed by p."@en ;
                  cco:ont00001760 "https://www.commoncoreontologies.org/InformationEntityOntology"^^xsd:anyURI .
 
 
 ###  https://www.commoncoreontologies.org/ont00001982
 cco:ont00001982 rdf:type owl:ObjectProperty ;
                  rdfs:subPropertyOf cco:ont00001808 ;
                  rdfs:domain cco:ont00000853 ;
+                 rdfs:range obo:BFO_0000001 ;
                  rdfs:comment "It is possible that this relation should be a functional property, that is for all x, y, z if x describes y and x describes z then y = z. For example, if a financial report x describes the quarterly results of a company y and that same financial report describes the quarterly results of a company z, then it should be inferred that companies y and z are the same. We refrained from classifying the relation as a functional property on the concern that descriptions are multifaceted and so consequently it may be that the same report would contain descriptions of multiple entities."@en ;
                  rdfs:label "describes"@en ;
-                 skos:definition "x describes y iff x is an instance of Information Content Entity, and y is an instance of Entity, such that x is about the characteristics by which y can be recognized or visualized."@en ;
+                 skos:definition "x describes y iff x is an instance of Descriptive Information Content Entity, and y is an instance of Entity, such that x is_about the characteristics that identify y."@en ;
```

Comment:
Identifying borders on designating. Maybe x describes y if f consists of sentences / propositions true of y.